<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogProcessingOptionsWidget</class>
 <widget class="QWidget" name="LogProcessingOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Log processing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QFrame" name="frmLogProcessing">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_34">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbLogProcessingBackup">
        <property name="text">
         <string>Use backup log when primary is not accessible</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbLogProcessingBoth">
        <property name="text">
         <string>Use primary and backup logs all the time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxLogProcessingSpecial">
     <property name="title">
      <string> Special file processing: </string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkLogProcessingCommon">
        <property name="text">
         <string>If common log(s)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblLogProcessingCommon">
        <property name="text">
         <string>execute command</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editLogProcessingCommonCmd">
        <property name="text">
         <string>cmd  /c  del  &quot;%log%&quot;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkLogProcessingPrivate">
        <property name="text">
         <string>If private log(s)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblLogProcessingPrivate">
        <property name="text">
         <string>execute command</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editLogProcessingPrivateCmd">
        <property name="text">
         <string>cmd  /c  move /y  &quot;%log%&quot;  &quot;%logpath%old\%logname%&quot;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cmbLogProcessingCommonCriteria">
          <item>
           <property name="text">
            <string>was created</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>is bigger than</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>is switching</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinLogProcessingCommonTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>48</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinLogProcessingCommonSize">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>10000000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>600000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbLogProcessingCommonTime">
          <item>
           <property name="text">
            <string>day(s) ago</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>week(s) ago</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>month(s) ago</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>year(s) ago</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblLogProcessingKb_2">
          <property name="text">
           <string>KB</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="cmbLogProcessingPrivateCriteria">
          <item>
           <property name="text">
            <string>was created</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>is bigger than</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>is switching</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinLogProcessingTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>48</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinLogProcessingSize">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>10000000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>200000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbLogProcessingTime">
          <item>
           <property name="text">
            <string>day(s) ago</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>week(s) ago</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>month(s) ago</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>year(s) ago</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblLogProcessingKb">
          <property name="text">
           <string>KB</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
